### ISSUE TITLE
Single click update of instanced properties back to source [3.2.0]

### ISSUE THREAD

**Issue by hlship** [godotengine/godot-proposals#905]
**Describe the project you are working on:**

2D arcade-like shooter.

**Describe the problem or limitation you are having in your project:**

It can be awkward to make changes to an instanced scene when you want to change the source scene, not the local instance.  This involves a lot of copying and clicking around to capture the changed properties, one-by-one, from the instanced scene into the original scene.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**

When using instanced scenes in the scene editor, sometimes I would like to make changes to properties (such as scale) and have an easy way to propagate that change backwards to the source scene.  It feels like this could be done with a new button or button behavior, rather than a multi-step manual process.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**

I would imagine that it could be implemented as an alt-click of the "reset instanced property" button that appears next to changed instance properties.

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**

I don't believe so. Perhaps my knowledge of Godot is too limited.

**Is there a reason why this should be core and not an add-on in the asset library?:**

This would be an editor and editor workflow change.

**Comment by hlship** [comment]
I did some searching to see if this is something that has already been suggested. My apologies if it has been and I was unable to find it.

**Comment by jonbonazza** [comment]
100% agree with this. This is one of the biggest PITAs when working with 3D in Godot.

**Comment by Calinou** [comment]
This is likely impossible to implement given how the editor currently works, since you can't edit the source scene without switching to it in the editor.

**Comment by Calinou** [comment]
@KoBeWi Do you see a way this could be implemented? If not, this proposal should be closed. It hasn't gathered much community support anyway.

**Comment by KoBeWi** [comment]
> This is likely impossible to implement given how the editor currently works

Not really. When you rename files, all dependent scenes are updated (I assume they are loaded temporarily and resaved). So this is possible.
